prog/hints.go

```diff
@@ -84,9 +84,86 @@ func generateHints(p *Prog, compMap CompMap, c *Call, arg Arg, exec func(newP *P
 }
 
 func checkConstArg(arg *ConstArg, compMap CompMap, cb func(newArg Arg)) {
-	for v, _ := range compMap[arg.Val] {
-		cb(constArg(arg.typ, v))
+	replacers := make(uint64Set)
+	for _, mutation := range mutations {
```

Comment:
mutations has just one element, and we don't have immediate plans to extend it. Just call shrinkExpand directly here. Bloating all code for any ever possible future extension is not a good idea. If/when we have a second mutation, it will be trivial to restore this code.
Also, then shrinkExpand can return replacers, and then we can do:
```
for replacer := range shrinkExpand(arg.Val, compMap) {
```